docs/self-managed/deployment/helm/configure/authentication-and-authorization/generic-oidc-provider.md

```diff
@@ -1,146 +1,784 @@
+            - admin@example.com
+        connectors:
+          clients:
+            - orchestration
+
+# Connectors
+connectors:
+  enabled: true
+  security:
+    authentication:
+      method: oidc
+      oidc:
+        clientId: orchestration
+        audience: orchestration
+        secret:
+          existingSecret: oidc-credentials
+          existingSecretKey: orchestration-client-secret
+
+# Management Identity
+identity:
+  enabled: true
+
+identityPostgresql:
+  enabled: true
+  auth:
+    existingSecret: camunda-credentials
+    secretKeys:
+      adminPasswordKey: identity-postgresql-admin-password
+      userPasswordKey: identity-postgresql-user-password
+
+# Disable internal Keycloak
+identityKeycloak:
+  enabled: false
+
+# Optimize
+optimize:
+  enabled: true
+
+# Web Modeler
+webModeler:
+  enabled: true
+  restapi:
+    mail:
+      fromAddress: noreply@example.com
+
+webModelerPostgresql:
+  enabled: true
+  auth:
+    existingSecret: camunda-credentials
+    secretKeys:
+      adminPasswordKey: webmodeler-postgresql-admin-password
+      userPasswordKey: webmodeler-postgresql-user-password
+
+# Console
+console:
+  enabled: true
 ```
 
-You can also [store the client secrets in a Kubernetes secret](/self-managed/deployment/helm/install/quick-install.md#create-identity-secrets) and reference this in the Helm values.
+**Placeholders to replace:**
 
-</TabItem>
-</Tabs>
+| Placeholder                                               | Replace With                                 |
+| --------------------------------------------------------- | -------------------------------------------- |
+| `https://your-provider.example.com`                       | Your OIDC provider's issuer URL              |
+| `identity`, `orchestration`, `optimize`, etc.             | Your actual client IDs                       |
+| `identity`, `orchestration`, `optimize` (audience values) | Actual audience values from token inspection |
+| `admin@example.com`                                       | Your admin user's claim value                |
 
-:::note
-Once set, you cannot update your initial claim name and value using environment or Helm values. You must change these values directly in the database.
-:::
+**Before deploying, verify:**
+
+- [ ] All `<placeholders>` replaced with actual values
+- [ ] All client secrets stored in the `oidc-credentials` secret
+- [ ] Database passwords stored in the `camunda-credentials` secret
+- [ ] Redirect URIs in OIDC provider match `redirectUrl` values
+- [ ] Token inspection confirms audience values
+- [ ] Verify tokens contain `preferred_username` and `client_id` claims, or uncomment and configure alternative claim names
+
+## Connect to the cluster
+
+After deploying Camunda with this configuration, use the following `kubectl port-forward` commands to access the APIs and UIs:
 
-<h3>Additional considerations</h3>
+```bash
+# Management Identity
+kubectl port-forward svc/camunda-identity 8084:80 -n camunda
```

Comment:
In the other guides these commands are without the namespace. Let's make it consistent.